Automattic/jetpack
# Update/jetpack settings connection nudges

## Proposed changes:

* Replace older connection nudge design with newer one for all Settings modules
Note: Except newsletter, that is handled here: https://github.com/Automattic/jetpack/pull/39539

### Other information:


## Jetpack product discussion

P2: pbNhbs-bHl-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. With a site connection only, go to `/wp-admin/admin.php?page=jetpack#/security`
3. Click through all the tabs and ensure all connection nudges have the same design. Below are screenshots of the updated ones
![image](https://github.com/user-attachments/assets/c9374e33-d5c0-4290-9581-a7c88ba2b21c)
![image](https://github.com/user-attachments/assets/dd657fd4-2cbb-41ac-9657-688d4f923819)
![image](https://github.com/user-attachments/assets/b723ebb4-b142-4fc9-8337-0555aa1664df)
![image](https://github.com/user-attachments/assets/7b6fc835-cd69-4996-aba4-3fb9f988073e)
![image](https://github.com/user-attachments/assets/10e7c4fe-6716-4bde-ac71-2e21219624e4)
![image](https://github.com/user-attachments/assets/79263b54-dec1-42ed-8fd2-b7b7ccbe8a22)
![image](https://github.com/user-attachments/assets/d97837c4-9851-4d41-86e3-0d21e52b2020)
